src/algorithms.jl

```diff
@@ -60,6 +60,7 @@ struct NDBLSRK134 <: OrdinaryDiffEqAlgorithm end
 struct NDBLSRK144 <: OrdinaryDiffEqAlgorithm end
 struct CFRLDDRK64 <: OrdinaryDiffEqAlgorithm end
 struct TSLDDRK74 <: OrdinaryDiffEqAlgorithm end
+struct CKLLDDRK43_2 <: OrdinaryDiffEqAdaptiveAlgorithm end
```

Comment:
I still think that this algorithm shouldn't be `CKLLDDRK43_2` because it is not LDD.

Reply:
I think it is a low storage algorithm but I'm not exactly sure about the dispersion and dissipation part, although I implemented it assuming it was an `LDDRK`

Reply:
In their article, Carpenter, Kennedy and Lewis derive algorithms for the Navier-Stokes equation. Thus, they try to find a reasonable balance between different properties. Nevertheless, their `C` schemes such as RK3(2)4[2R+]C are optimised for error efficiency, not for dispersion & dissipation.